src/mixins/child-apps.js

```diff
@@ -73,11 +81,12 @@ export default {
    * @method _startChildApps
    */
   _startChildApps() {
-    _.each(this._childApps, function(childApp) {
-      if(_.result(childApp, 'startWithParent')) {
+    const isRestarting = this._isRestarting;
```

Comment:
nope do the check itself outside of the loop  `const shouldStartOption = this._isRestarting ? 'restartWithParent' : 'startWithParent'`